mms/service.py

```diff
@@ -112,9 +112,9 @@ def predict(self, batch):
         except MemoryError:
             logger.error("System out of memory", exc_info=True)
             return create_predict_response(None, req_id_map, "Out of resources", 507)
-        except Exception:  # pylint: disable=broad-except
+        except Exception as e:  # pylint: disable=broad-except
             logger.warning("Invoking custom service failed.", exc_info=True)
-            return create_predict_response(None, req_id_map, "Prediction failed", 503)
+            return create_predict_response(None, req_id_map, "Prediction failed : [{}]".format(e), 503)
```

Comment:
No, this is bad practice to return exception details to client for unknown generic exceptions, as it can expose sensitive implementation details.

It would be fine to log the original exception though when wrapping in generic message.

Even without patch, the model code can use `mms.PredictionException` to preserve error code and message (see line 109 just a few lines above this)